dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.telemetry-1.0.feature

```diff
@@ -0,0 +1,9 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.org.eclipse.microprofile.telemetry-1.0
+visibility=private
+singleton=true
+-features=io.openliberty.mpCompatible-5.0
+-bundles=io.openliberty.org.eclipse.microprofile.telemetry.1.0; location:="dev/api/stable/,lib/"
+kind=ga
```

Comment:
We should check this in initially as `noship` first and change it to `beta` after all tests passed.